### DISCUSSION TITLE
I have a questions about the ingredients table?

### DISCUSSION THREAD

**Discussion by agnusdei13** [MIT-LCP/mimic-code#1518] [MIMIC-IV] [answered]
Thank you for your great job sharing the dataset.
I'm currently analyzing clinical outcomes with the mimic-iv dataset.
During exploring data, I have a question about the meaning of each variable in the ingredients tables.
In MICU cohorts, I have the following results after querying and refining the process with Jupyter-lab.

H2O                                288090
IVF ingredient                     199481
Calories                           100981
Oral/Gastric Ingredient             63771
Protein                             16812
Enteral Nutrition Ingredient        16679
Blood Ingredient                     6394
Ca (ingr)                            6144
Carb's                                689
Glucose (ingr)                        689
Kcal                                  689
KJ                                    689
Parenteral Nutrition Ingredient       441
OR Intake_ingr                        271
Supplement Ingredient                 114
PACU Intake_ingr                        7

Can you please tell me what these variables mean or how they are created?
For example, H2O means the water intake by patients? orally or parenterally?
Does 'Caroies' mean the calorie intake by each record?
How can I estimate the daily intake of one patient? Simply sum of all these variables? 
H2O and IV ingredients look similar to me and the counts of 'Parenteral Nutrition Ingredient' is too small compared to other Ingredients such as 'Enteral Nutrition Ingredient'.

**Comment by alistairewj** [comment, marked as answer]
> Can you please tell me what these variables mean or how they are created? For example, H2O means the water intake by patients? orally or parenterally?

This depends on the associated input, which you can link to via `orderid`.

> Does 'Calories' mean the calorie intake by each record?

Yes.

> How can I estimate the daily intake of one patient? Simply sum of all these variables? H2O and IV ingredients look similar to me and the counts of 'Parenteral Nutrition Ingredient' is too small compared to other Ingredients such as 'Enteral Nutrition Ingredient'.

I would sum *only* the water to estimate the total fluid intake of each patient. Here is a diagram from the original user manual of the collection software:

<img width="616" alt="Screen Shot 2023-04-10 at 7 59 51 AM" src="https://user-images.githubusercontent.com/1282676/230897686-b8a71b23-abb2-4b90-b088-99f1b5151cb0.png">

The reason why *ingredientevents* was added was basically so that this calculation could be carried out.

**Comment by agnusdei13** [comment]
Thank you for your kind reply. I explored the database again, I understood what you mean.
